src/classes/PMT_Payment_TEST.cls

```diff
@@ -381,6 +381,234 @@ private class PMT_Payment_TEST {
     /*******************************************************************************************************
      * HELPER FUNCTIONS
      *******************************************************************************************************/
+    /**
+     * @description Validate that the accounting rules are enforced properly. If there is a paid payment, it must have a payment date.
+     * If it is an unpaid payment, it must have a scheduled date. 
+     * down the Payment records.
+     */
+     @isTest
+    static void test_EnforceAccountingDataConsistency() {
```

Comment:
The test method shouldn't be below the comment block that indicates methods below it are for `helper` methods.

Reply:
Agreed, will change now.